src/box/raft.c

```diff
@@ -301,6 +302,24 @@ box_raft_fence(void)
 	raft_resign(raft);
 }
 
+/**
+ * Resign RAFT leadership and freeze limbo regardless of
+ * box_election_fencing_mode. It waits until the elections
+ * begin. After the death-timeout expires, it starts a new
+ * round of elections.
+ */
+static void
+box_raft_leader_step_off(void)
+{
+	struct raft *raft = box_raft();
+	if (!raft->is_enabled || raft->state != RAFT_STATE_LEADER)
+		return;
+
+	txn_limbo_fence(&txn_limbo);
```

Comment:
Who is going to unfence it later? Please, leave a comment.

Reply:
Done.